package not working with typescript@3.0.1

The global declaration of Element {
  host: any
}

is conflicting with the new typescript version(3.0.1) where host is of string type.

Getting the following errors while using this package

/node_modules/typescript/lib/lib.dom.d.ts:5547:11 - error TS2320: Interface 'HTMLAnchorElement' cannot simultaneously extend types 'HTMLElement' and 'HTMLHyperlinkElementUtils'.
  Named property 'host' of types 'HTMLElement' and 'HTMLHyperlinkElementUtils' are not identical.

/node_modules/typescript/lib/lib.dom.d.ts:5658:11 - error TS2320: Interface 'HTMLAreaElement' cannot simultaneously extend types 'HTMLElement' and 'HTMLHyperlinkElementUtils'.
  Named property 'host' of types 'HTMLElement' and 'HTMLHyperlinkElementUtils' are not identical.

--- a/src/index.ts
+++ b/src/index.ts
@@ -22,11 +22,6 @@ declare global {
       width: number
     }
   }
-
-  // @TODO better declaration of possible shadowdom hosts
-  interface Element {
-    host: any
-  }
 }
 
 type ScrollLogicalPosition = 'start' | 'center' | 'end' | 'nearest'

--- a/src/index.ts
+++ b/src/index.ts
@@ -53,7 +53,7 @@ function isElement(el: any) {
   return (
     el != null &&
     typeof el === 'object' &&
-    (el.nodeType === 1 || el.nodeType === 11)
+    (el.nodeType === 1)
   )
 }
 

--- a/src/index.ts
+++ b/src/index.ts
@@ -247,7 +247,7 @@ export default (target: Element, options: Options): CustomScrollAction[] => {
   let cursor = target
   while (isElement(cursor) && checkBoundary(cursor)) {
     // Move cursor to parent or shadow dom host
-    cursor = cursor.parentNode || (cursor as HTMLElement).host
+    cursor = cursor.parentNode || (cursor as ShadowRoot).host
 
     // Stop when we reach the viewport
     if (cursor === scrollingElement) {

--- a/src/index.ts
+++ b/src/index.ts
@@ -247,7 +247,7 @@ export default (target: Element, options: Options): CustomScrollAction[] => {
   let cursor = target
   while (isElement(cursor) && checkBoundary(cursor)) {
     // Move cursor to parent or shadow dom host
-    cursor = cursor.parentNode || cursor.host
+    cursor = cursor.parentNode || (cursor as HTMLElement).host
 
     // Stop when we reach the viewport
     if (cursor === scrollingElement) {

--- a/src/index.ts
+++ b/src/index.ts
@@ -50,11 +50,7 @@ interface CustomScrollAction {
 
 // @TODO better shadowdom test, 11 = document fragment
 function isElement(el: any) {
-  return (
-    el != null &&
-    typeof el === 'object' &&
-    (el.nodeType === 1)
-  )
+  return el != null && typeof el === 'object' && el.nodeType === 1
 }
 
 function canOverflow(

--- a/src/index.ts
+++ b/src/index.ts
@@ -246,8 +246,8 @@ export default (target: Element, options: Options): CustomScrollAction[] => {
   const frames: Element[] = []
   let cursor = target
   while (isElement(cursor) && checkBoundary(cursor)) {
-    // Move cursor to parent or shadow dom host
-    cursor = cursor.parentNode || (cursor as ShadowRoot).host
+    // Move cursor to parent
+    cursor = cursor.parentNode
 
     // Stop when we reach the viewport
     if (cursor === scrollingElement) {

--- a/src/index.ts
+++ b/src/index.ts
@@ -243,7 +243,7 @@ export default (target: Element, options: Options): CustomScrollAction[] => {
   let cursor = target
   while (isElement(cursor) && checkBoundary(cursor)) {
     // Move cursor to parent
-    cursor = cursor.parentNode
+    cursor = cursor.parentNode as Element
 
     // Stop when we reach the viewport
     if (cursor === scrollingElement) {

